projects/ag-grid/documentation/code-examples/data-grid/data-manager-added/data-manager-data-grid-docs-demo.component.ts

```diff
@@ -0,0 +1,376 @@
+  SkyDataViewConfig,
+} from '@skyux/data-manager';
+
+import {
+  GridApi,
+  GridReadyEvent,
+  GridOptions,
+  ValueFormatterParams,
+  ColumnApi,
+  ColDef,
+} from 'ag-grid-community';
+
+import { SKY_AG_GRID_DEMO_DATA } from './data-manager-data-grid-docs-demo-data';
+import { DataManagerDataGridDocsDemoFiltersModalComponent } from './data-manager-data-grid-docs-demo-filter-modal.component';
+
+@Component({
+  selector: 'app-data-manager-data-grid-docs-demo',
+  templateUrl: './data-manager-data-grid-docs-demo.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SkyDataManagerDataGridDemoComponent implements OnInit {
+  public columnDefs: ColDef[] = [
+    {
+      field: 'selected',
+      type: SkyCellType.RowSelector,
+    },
+    {
+      field: 'name',
+      headerName: 'Name',
+    },
+    {
+      field: 'age',
+      headerName: 'Age',
+      type: SkyCellType.Number,
+      maxWidth: 60,
+    },
+    {
+      field: 'startDate',
+      headerName: 'Start Date',
+      type: SkyCellType.Date,
+      sort: 'asc',
+    },
+    {
+      field: 'endDate',
+      headerName: 'End Date',
+      type: SkyCellType.Date,
+      valueFormatter: this.endDateFormatter,
+    },
+    {
+      field: 'department',
+      headerName: 'Department',
+      type: SkyCellType.Autocomplete,
+    },
+    {
+      field: 'jobTitle',
+      headerName: 'Title',
+      type: SkyCellType.Autocomplete,
+    },
+    {
+      colId: 'validationCurrency',
+      field: 'validationCurrency',
+      type: [SkyCellType.CurrencyValidator],
+    },
+    {
+      colId: 'validationDate',
+      field: 'validationDate',
+      type: [SkyCellType.Date, SkyCellType.Validator],
+      cellRendererParams: {
+        skyComponentProperties: {
+          validator: (value: Date) => !!value && value > new Date(1985, 9, 26),
+          validatorMessage: 'Please enter a future date',
+        },
+      },
+    },
+  ];
+
+  public columnApi?: ColumnApi;
+  public displayedItems: any[] = [];
+  public gridApi?: GridApi;
+  public gridInitialized = false;
+  public items = SKY_AG_GRID_DEMO_DATA;
+  public gridOptions!: GridOptions;
+
+  public searchText: string = '';
+  public activeViewId = 'dataManagerGridView';
+  public viewId = 'dataManagerGridView';
+  public viewConfig: SkyDataViewConfig = {
+    id: this.viewId,
+    name: 'Grid View',
+    icon: 'table',
+    searchEnabled: true,
+    sortEnabled: true,
+    multiselectToolbarEnabled: true,
+    columnPickerEnabled: true,
+    filterButtonEnabled: true,
+    columnOptions: [
+      {
+        id: 'selected',
+        label: 'selected',
+        alwaysDisplayed: true,
+      },
+      {
+        id: 'name',
+        label: 'Name',
+        description: 'The name of the employee.',
+      },
+      {
+        id: 'age',
+        label: 'Age',
+        description: 'The age of the employee.',
+      },
+      {
+        id: 'startDate',
+        label: 'Start Date',
```

Comment:
Sentence-style capitalization...
```suggestion
        label: 'Start date',
```